dts/bindings/ethernet/atmel,sam-gmac.yaml

```diff
@@ -15,47 +15,47 @@ properties:
     num-queues:
       type: int
       required: true
-      description: Number of hardware TX and RX queues
+      description: |
+        Number of hardware TX and RX queues.
 
     max-frame-size:
-        type: int
-        description:
-          Maximum transfer unit (IEEE defined MTU), rather than the
-          maximum frame size (there\'s contradiction in the Devicetree
-          Specification). The current supported values are 1518, 1536
-          and 10240 (jumbo frames).
-        default: 1518
+      type: int
+      default: 1518
+      description: |
+        Maximum transfer unit (IEEE defined MTU), rather than the maximum
+        frame size (there\'s contradiction in the Devicetree Specification).
+        The current supported values are 1518, 1536 and 10240 (jumbo frames).
 
     max-speed:
-        type: int
-        description:
-          Specifies maximum speed in Mbit/s supported by the device.
-        default: 100
+      type: int
+      default: 100
```

Comment:
Same here; why is this the default? Can you justify it if you have time? Again not blocking since the problem existed before this PR.